### DISCUSSION TITLE
lotus-miner actor set-addrs not work

### DISCUSSION THREAD

**Discussion by seantw1974** [filecoin-project/lotus#8327] [General Q&A] [answered]
I change my public IP.
And then I use lotus-miner actor set-addrs to setup new ip
but in filscan the IP always the old one !
I'v wait around 12hours...
and same same ...
any suggestion ?
https://filscan.io/statistics/peer?peer_id=12D3KooWJ19HVCXXqWKuJ4MPFt83eUuf6mHj1MyJGr2wbsKztDNS

**Comment by rjan90** [comment, marked as answer]
Hey @seantw1974.

Did you set your actor address with the: `lotus-miner actor set-addrs /ip4/<YOUR_PUBLIC_IP_ADDRESS>/tcp/24001` format?

Looking on-chain it seems like you had a couple of successful `ChangeMultiaddrs` messages. Did the issue get resolved? The messages might have been stuck in your message pool.

**Reply by seantw1974** [reply]
it's seems "filscan" website issue
Thank you

**Reply by rjan90** [reply]
Good to hear! Closing this issue now then.

Thank you for the report!
